snooty/rstspec.toml

```diff
@@ -641,6 +641,53 @@ example = """.. card-group::
    ${0: Card content}
 """
 
+[directive."landing:product"]
+content_type = "block"
+help = """Product in a product-list. Each product links to the specified URI, and can include content, an icon, as well as a call-to-action string"""
+options.icon = ["path", "uri"]
+options.icon-alt = "string"
+example = """.. product::
+   ${4::icon: /path-to-icon.svg
+   :icon-alt: Icon's alt text
+
+   ${0:Product content.}
+"""
+
+[directive."landing:products"]
+content_type = "block"
+help = """Container to hold a group of products"""
+example = """.. landing:products::
+   ${0: Product content}
+"""
+
+[directive."landing:client-libraries"]
+content_type = "block"
+help = """Title and call-to-action for drivers card"""
+example = """.. landing:client-libraries::
+   ${0: Library content}
+"""
+
+[directive."landing:introduction"]
+content_type = "block"
```

Comment:
similar here as above. [the heading](https://github.com/schmalliso/docs-landing/blob/3099d7975ecfa8f1f018034b35ba7b8a00af5416/source/index.txt#L24) is being treated as as a paragraph

```
{
  "type": "paragraph",
  "position": {
    "start": {
      "line": {
        "$numberInt": "23"
      }
    }
  },
  "children": [
    {
      "type": "text",
      "position": {
        "start": {
          "line": {
            "$numberInt": "23"
          }
        }
      },
      "value": "Developer Data Platform"
    }
  ]
}
```

Reply:
@seungpark same as above, this is a placeholder as well. This task is asked in a different ticket. I can update this all in this ticket, and update the ticket as well if you prefer.  I just used these as placeholders to get the parser to successfully run.